README.rst

```diff
@@ -11,6 +11,11 @@ Installing
 ----------
 `Here <https://github.com/mathics/Mathics/wiki/Installing>`_ is a short guide on how to install Mathics on your computer.
 
+Docker
```

Comment:
This is completely unrelated to the PR. If you'd like to address this please create a separate PR.